{"totalCount":5,"grade":"D","repositories":[{"url":"https://github.com/MTES-MCT/sparte","vulnerabilityAlerts":{"totalCount":5,"nodes":[{"dismissedAt":null,"createdAt":"2022-10-18T21:28:30Z","securityVulnerability":{"severity":"MODERATE","package":{"name":"py"},"advisory":{"identifiers":[{"type":"GHSA","value":"GHSA-w596-4wvx-j9j6"},{"type":"CVE","value":"CVE-2022-42969"}],"references":[{"url":"https://nvd.nist.gov/vuln/detail/CVE-2022-42969"},{"url":"https://github.com/pytest-dev/py/issues/287"},{"url":"https://github.com/pytest-dev/py/blob/cb87a83960523a2367d0f19226a73aed4ce4291d/py/_path/svnurl.py#L316"},{"url":"https://github.com/pytest-dev/py/issues/288"},{"url":"https://github.com/pytest-dev/pytest/issues/10392"},{"url":"https://news.ycombinator.com/item?id=34163710"},{"url":"https://pypi.org/project/py"},{"url":"https://github.com/advisories/GHSA-w596-4wvx-j9j6"}]}}},{"dismissedAt":null,"createdAt":"2023-03-15T11:25:46Z","securityVulnerability":{"severity":"HIGH","package":{"name":"webpack"},"advisory":{"identifiers":[{"type":"GHSA","value":"GHSA-hc6q-2mpp-qw7j"},{"type":"CVE","value":"CVE-2023-28154"}],"references":[{"url":"https://nvd.nist.gov/vuln/detail/CVE-2023-28154"},{"url":"https://github.com/webpack/webpack/pull/16500"},{"url":"https://github.com/webpack/webpack/compare/v5.75.0...v5.76.0"},{"url":"https://github.com/advisories/GHSA-hc6q-2mpp-qw7j"}]}}},{"dismissedAt":null,"createdAt":"2023-03-27T22:07:30Z","securityVulnerability":{"severity":"HIGH","package":{"name":"redis"},"advisory":{"identifiers":[{"type":"GHSA","value":"GHSA-24wv-mv5m-xv4h"},{"type":"CVE","value":"CVE-2023-28858"}],"references":[{"url":"https://nvd.nist.gov/vuln/detail/CVE-2023-28858"},{"url":"https://github.com/redis/redis-py/issues/2624"},{"url":"https://github.com/redis/redis-py/pull/2641"},{"url":"https://github.com/redis/redis-py/compare/v4.3.5...v4.3.6"},{"url":"https://github.com/redis/redis-py/compare/v4.4.2...v4.4.3"},{"url":"https://github.com/redis/redis-py/compare/v4.5.2...v4.5.3"},{"url":"https://openai.com/blog/march-20-chatgpt-outage"},{"url":"https://github.com/redis/redis-py/releases/tag/v4.4.4"},{"url":"https://github.com/redis/redis-py/releases/tag/v4.5.4"},{"url":"https://github.com/advisories/GHSA-24wv-mv5m-xv4h"}]}}},{"dismissedAt":null,"createdAt":"2023-03-31T14:59:56Z","securityVulnerability":{"severity":"MODERATE","package":{"name":"redis"},"advisory":{"identifiers":[{"type":"GHSA","value":"GHSA-8fww-64cx-x8p5"},{"type":"CVE","value":"CVE-2023-28859"}],"references":[{"url":"https://nvd.nist.gov/vuln/detail/CVE-2023-28859"},{"url":"https://github.com/redis/redis-py/issues/2665"},{"url":"https://github.com/redis/redis-py/pull/2641"},{"url":"https://github.com/redis/redis-py/pull/2666"},{"url":"https://github.com/redis/redis-py/pull/2671"},{"url":"https://github.com/redis/redis-py/releases/tag/v4.4.4"},{"url":"https://github.com/redis/redis-py/releases/tag/v4.5.4"},{"url":"https://github.com/advisories/GHSA-8fww-64cx-x8p5"}]}}},{"dismissedAt":null,"createdAt":"2023-04-21T22:32:38Z","securityVulnerability":{"severity":"MODERATE","package":{"name":"sqlparse"},"advisory":{"identifiers":[{"type":"GHSA","value":"GHSA-rrm6-wvj7-cwh2"},{"type":"CVE","value":"CVE-2023-30608"}],"references":[{"url":"https://github.com/andialbrecht/sqlparse/security/advisories/GHSA-rrm6-wvj7-cwh2"},{"url":"https://nvd.nist.gov/vuln/detail/CVE-2023-30608"},{"url":"https://github.com/andialbrecht/sqlparse/commit/c457abd5f097dd13fb21543381e7cfafe7d31cfb"},{"url":"https://github.com/andialbrecht/sqlparse/commit/e75e35869473832a1eb67772b1adfee2db11b85a"},{"url":"https://owasp.org/www-community/attacks/Regular_expression_Denial_of_Service_-_ReDoS"},{"url":"https://github.com/advisories/GHSA-rrm6-wvj7-cwh2"}]}}}]},"grade":"D"}]}